docs/pages/getting-started/install.md

```diff
@@ -1,141 +1,343 @@
+
+    ```bash
+    pip install kedro
+    ```
+
+    You can use any modern Python packaging tool you want, such as Poetry, PDM, or Hatch:
+
+    ```bash
+    poetry add kedro
+    ```
+
+=== "conda"
+
+    You can also install Kedro using `conda` as follows:
+
+    ```bash
+    conda install -c conda-forge kedro
+    ```
+
+### How to manually install the project dependencies in an environment
+
+Kedro projects have a `pyproject.toml` that specifies their dependencies
+and enable sharable projects by ensuring consistency across Python packages and versions.
+A `requirements.txt` file is also included for compatibility with legacy workflows.
+
+This is a sample of the dependencies found in the Spaceflights Kedro project
+(you may find that the versions differ slightly depending on the version of Kedro):
+
+```text
+# code quality packages
+ipython~=8.10; python_version >= '3.8'
+ruff==0.1.8
+
+# notebook tooling
+jupyter~=1.0
+jupyterlab_server>=2.11.1
+jupyterlab~=3.0
+
+# Pytest + useful extensions
+pytest-cov~=3.0
+pytest-mock>=1.7.1, <2.0
+pytest~=7.2
+
+# Kedro dependencies and datasets to work with different data formats (including CSV, Excel, and Parquet)
+kedro~=0.19.0
+kedro-datasets[pandas-csvdataset, pandas-exceldataset, pandas-parquetdataset]>=3.0
+kedro-telemetry>=0.3.1
+kedro-viz~=6.0 # Visualise pipelines
+
+# For modeling in the data science pipeline
+scikit-learn~=1.0
 ```
 
-When migrating an existing project to a newer Kedro version, make sure you also update the `kedro_init_version`:
+=== "uv"
+
+    You can use `uv` to sync the environment with the project dependencies:
+
+    ```bash
+    uv sync
+    ```
+
+    Alternatively, you can use the lower level interface to install the dependencies
+    directly from the `requirements.txt` file:
+
+    ```bash
+    uv pip install -r requirements.txt
+    ```
+
+=== "Other native tools"
+
+    You can use pip or similar tools to install the dependencies
+    directly from the `requirements.txt` file:
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+=== "conda"
+
+    !!! warning
+
+        The official Kedro starters don't ship with `environment.yml` files
+        that can be consumed by `conda`.
+        Use `uv` or other native tools instead.
+
+## Other setup steps
+
+### How to integrate Kedro in VS Code with the official extension
+Working in an IDE can be a great productivity boost.
+
+For VS Code Users: Check out [Set up Visual Studio Code](../ide/set_up_vscode.md) and [Kedro VS Code Extension](../ide/set_up_vscode.md#kedro-vs-code-extension).
+
+For PyCharm Users: Checkout [Set up PyCharm](../ide/set_up_pycharm.md).
+
+### How to upgrade Kedro
+
+The best way to safely upgrade is to check our [release notes](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) for any notable breaking changes. Follow the steps in the migration guide included for that specific release.
```

Comment:
Maybe clarify that upgrades within a series and between series is different? Usually the first doesn't include a migration guide.

Reply:
Good catch as well, will do